### PR TITLE
Add Instant Gaming

### PR DESCRIPTION
**Pull request by Machou**
No Doc for 2FA on the setup, see below

Sans titre-2

### CODE REVIEW

**Comment by Machou** [comment]
Sorry, I meant to resume on the other PR, but I don't know why I couldn't edit the file anymore.

File: img/i/instant-gaming.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...ff7207"/><stop offset="1" stop-color="#ff7208" stop-opacity="0"/></linearGradient><radialGradient id="C" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(31.616573343087463,1.5055616231730151,-1.6275956206080497,34.17926939675244,34.6276,229.717)"><stop stop-color="#ff7307"/><stop offset="1" stop-color="#ff7408" stop-opacity="0"/></radialGradient><linearGradient id="D" x1="13.055" y1="376" x2="349.288" y2="170.804" gradientUnits="userSpaceOnUse"><stop stop-color="#ff8000"/><stop offset="1" stop-color="#ff4020"/></linearGradient><clipPath id="E"><path fill="#fff" d="M0 0h1458v376H0z"/></clipPath><path id="F" d="M294.137 177.33L36.769 325.812v-63.635l201.909-116.553-36.583-21.106L9.194 235.663c-5.688 3.281-9.192 9.345-9.192 15.908V357.63a18.37 18.37 0 0 0 9.192 15.909A18.38 18.38 0 0 0 18.386 376a18.39 18.39 0 0 0 9.192-2.461l312.519-180.3c5.688-3.281 9.192-9.346 9.192-15.909s-3.504-12.627-9.192-15.908l-91.605-52.85a17.54 17.54 0 0 0-.594-.342l-91.635-52.867c-5.688-3.281-12.695-3.281-18.383 0L9.195 129.604c-8.793 5.073-11.805 16.307-6.729 25.094 3.405 5.893 9.584 9.188 15.938 9.188a18.32 18.32 0 0 0 9.174-2.464L147.07 92.484l147.067 84.847z"/></defs></svg>
```
Comment (fletcherdodd) [inline img/i/instant-gaming.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 349.29 376.13"><defs><linearGradient id="a" x1="2.39" x2="60.44" y1="78.75" y2="-5.3" gradientUnits="userSpaceOnUse"><stop offset=".26" stop-color="#ff670d"/><stop offset="1" stop-color="#ff5317"/></linearGradient><linearGradient id="c" x1="121.32" x2="197.09" y1="254.49" y2="122.43" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ff7207"/><stop offset="1" stop-color="#ff7208" stop-opacity="0"/></linearGradient><linearGradient id="e" x1="-38.51" x2="297.72" y1="291.64" y2="86.44" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ff8000"/><stop offset="1" stop-color="#ff4020"/></linearGradient><radialGradient id="d" cx="-222.21" cy="719.2" r="1" fx="-222.21" fy="719.2" gradientTransform="matrix(31.61408 1.50747 1.62988 -34.18116 5889.86 25146.2)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ff7307"/><stop offset="1" stop-color="#ff7408" stop-opacity="0"/></radialGradient><path id="b" d="M294.14 177.46 36.77 325.94V262.3l201.91-116.55-36.58-21.11L9.19 235.79A18.39 18.39 0 0 0 0 251.7v106.06c0 6.56 3.51 12.63 9.19 15.91a18.39 18.39 0 0 0 18.38 0l312.52-180.3a18.36 18.36 0 0 0 0-31.82l-91.61-52.85-.59-.34-91.64-52.87a18.4 18.4 0 0 0-18.38 0L9.19 129.73a18.36 18.36 0 0 0 9.21 34.28c3.22 0 6.39-.85 9.17-2.46l119.49-68.94 147.07 84.85Z"/></defs><path fill="url(#a)" d="M18.4 57.96a18.38 18.38 0 0 1-9.21-34.28L45.96 2.46a18.4 18.4 0 0 1 25.11 6.72 18.36 18.36 0 0 1-6.73 25.09L27.58 55.48a18.35 18.35 0 0 1-9.18 2.46"/><use xlink:href="#b" fill="url(#c)"/><use xlink:href="#b" fill="url(#d)"/><path fill="url(#e)" d="M294.14 177.46 36.77 325.94V262.3l11.64-6.72-20.83-30.39L9.2 235.78A18.4 18.4 0 0 0 0 251.7v106.06c0 6.56 3.51 12.63 9.19 15.91a18.39 18.39 0 0 0 18.38 0l312.52-180.3a18.36 18.36 0 0 0 0-31.82l-91.61-52.85-.59-.34-91.64-52.87a18.4 18.4 0 0 0-18.38 0L9.19 129.73a18.36 18.36 0 0 0 9.21 34.28c3.22 0 6.39-.85 9.17-2.46l119.49-68.94 147.07 84.85Z"/></svg>
```

Comment (fletcherdodd) [reply img/i/instant-gaming.com.svg]: Nice work on the pull request @Machou. I have modified the SVG to remove the name, [as you can see](https://2fa.directory/#gaming) with most other listing if the logo has a icon it preferred to use use that over the entire logo.

Couple of other suggestions for future PRs;
- If the site you are looking to add has no documenation listed, I always try to include a screenshot of the login and/or 2fa setup process. One here it clearly shows the methods you're proposing they support is most helpful.
- Secondly, with the SVGs, I always try to run them through a few different optimising tools as stated in the [contributing guide](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#guidelines) (under the guidelines section). This just ensures the file has only what it needs to display correctly to keep site loading efficient.
- If you need to make adjustments to a branch after you have comitted/pushed it, I always update the summary field from it's default suggestion to a _very_ brief description of what you are adjusting. It's not a major thing, just helpful for those reviewing the PR
- Lastly, for your convience (not a rule or anything), I would suggest creating branches for each PR you intend on raising. This just makes it easier to switch between them if you are working on multiple site updates, and if you break something on your local version you can just trim the branch and start again.

Keep up the good work though 😊

Comment (Machou) [reply img/i/instant-gaming.com.svg]: Ok, about the logo I wasn't sure how to go about it, either the icon with or without text, so I preferred to keep the original, and pass it into svgo (installed on my vm for the occasion).

Instant-Gaming didn't have a guid for 2FA, and I didn't think to take a screenshot ^^

Ok for the branch.

Thank you for your patience and advice :)

File: img/i/instant-gaming.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...trix(31.61408 1.50747 1.62988 -34.18116 5889.86 25146.2)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ff7307"/><stop offset="1" stop-color="#ff7408" stop-opacity="0"/></radialGradient><path id="b" d="M294.14 177.46 36.77 325.94V262.3l201.91-116.55-36.58-21.11L9.19 235.79A18.39 18.39 0 0 0 0 251.7v106.06c0 6.56 3.51 12.63 9.19 15.91a18.39 18.39 0 0 0 18.38 0l312.52-180.3a18.36 18.36 0 0 0 0-31.82l-91.61-52.85-.59-.34-91.64-52.87a18.4 18.4 0 0 0-18.38 0L9.19 129.73a18.36 18.36 0 0 0 9.21 34.28c3.22 0 6.39-.85 9.17-2.46l119.49-68.94 147.07 84.85Z"/></defs><path fill="url(#a)" d="M18.4 57.96a18.38 18.38 0 0 1-9.21-34.28L45.96 2.46a18.4 18.4 0 0 1 25.11 6.72 18.36 18.36 0 0 1-6.73 25.09L27.58 55.48a18.35 18.35 0 0 1-9.18 2.46"/><use xlink:href="#b" fill="url(#c)"/><use xlink:href="#b" fill="url(#d)"/><path fill="url(#e)" d="M294.14 177.46 36.77 325.94V262.3l11.64-6.72-20.83-30.39L9.2 235.78A18.4 18.4 0 0 0 0 251.7v106.06c0 6.56 3.51 12.63 9.19 15.91a18.39 18.39 0 0 0 18.38 0l312.52-180.3a18.36 18.36 0 0 0 0-31.82l-91.61-52.85-.59-.34-91.64-52.87a18.4 18.4 0 0 0-18.38 0L9.19 129.73a18.36 18.36 0 0 0 9.21 34.28c3.22 0 6.39-.85 9.17-2.46l119.49-68.94 147.07 84.85Z"/></svg>
```
Comment (hkamran80) [inline img/i/instant-gaming.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 349.3 376.1"><defs><linearGradient id="a" x1="2.4" x2="60.4" y1="78.8" y2="-5.3" gradientUnits="userSpaceOnUse"><stop offset=".3" stop-color="#ff670d"/><stop offset="1" stop-color="#ff5317"/></linearGradient><linearGradient id="c" x1="121.3" x2="197.1" y1="254.5" y2="122.4" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ff7207"/><stop offset="1" stop-color="#ff7208" stop-opacity="0"/></linearGradient><linearGradient id="e" x1="-38.5" x2="297.7" y1="291.6" y2="86.4" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ff8000"/><stop offset="1" stop-color="#ff4020"/></linearGradient><radialGradient id="d" cx="-222.2" cy="719.2" r="1" fx="-222.2" fy="719.2" gradientTransform="matrix(32 2 2 -34 5889.9 25146.2)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#ff7307"/><stop offset="1" stop-color="#ff7408" stop-opacity="0"/></radialGradient><path id="b" d="M294.1 177.5 36.8 325.9v-63.6l201.9-116.5-36.6-21.2-193 111.2A18.4 18.4 0 0 0 0 251.7v106c0 6.6 3.5 12.7 9.2 16a18.4 18.4 0 0 0 18.4 0L340 193.4a18.4 18.4 0 0 0 0-31.9l-91.6-52.8-.6-.3-91.7-53a18.4 18.4 0 0 0-18.3 0L9.2 129.8a18.4 18.4 0 0 0 9.2 34.3c3.2 0 6.4-.8 9.2-2.5L147 92.6l147 84.9Z"/></defs><path fill="url(#a)" d="M18.4 58a18.4 18.4 0 0 1-9.2-34.3L46 2.5a18.4 18.4 0 0 1 25 6.7 18.4 18.4 0 0 1-6.7 25L27.6 55.6a18.4 18.4 0 0 1-9.2 2.4"/><use xlink:href="#b" fill="url(#c)"/><use xlink:href="#b" fill="url(#d)"/><path fill="url(#e)" d="M294.1 177.5 36.8 325.9v-63.6l11.6-6.7-20.8-30.4-18.4 10.6A18.4 18.4 0 0 0 0 251.7v106c0 6.6 3.5 12.7 9.2 16a18.4 18.4 0 0 0 18.4 0L340 193.4a18.4 18.4 0 0 0 0-31.9l-91.6-52.8-.6-.3-91.7-53a18.4 18.4 0 0 0-18.3 0L9.2 129.8a18.4 18.4 0 0 0 9.2 34.3c3.2 0 6.4-.8 9.2-2.5L147 92.6l147 84.9Z"/></svg>
```

Here's a more minified SVG

File: entries/i/instant-gaming.com.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "Instant-Gaming": {
+    "domain": "instant-gaming.com",
+    "tfa": [
+      "totp"
```
Comment (hkamran80) [inline entries/i/instant-gaming.com.json:5]: Please attach some screenshots of the 2FA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/i/instant-gaming.com.json (hunk 3 above)
Comment (Machou) [reply entries/i/instant-gaming.com.json:5]: Of course, can you give me a commit or other to have an example. Because I didn't find an example ^^

Comment (fletcherdodd) [reply entries/i/instant-gaming.com.json:5]: [Here is a PR](https://github.com/2factorauth/twofactorauth/pull/6401) where I have included a screenshot of the login screen asking for a code, the screenshot shows SMS verification which matches what I put in the JSON.

Secondly, [here is a recent one](https://github.com/2factorauth/twofactorauth/pull/8034) where I have included both the setting in the account settings and what you see when you login. The bottom-right screenshot confirms its a code emailed to you.

I have blurred any sensitive information before attaching these images.

Comment (Machou) [reply entries/i/instant-gaming.com.json:5]: @fletcherdodd i have add image how to setup 2fa, i hope it's ok :)

Comment (fletcherdodd) [reply entries/i/instant-gaming.com.json:5]: That should be more than adequate! Nice work.

**Review by Carlgo11 — APPROVED** [review]

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**